features/step_definitions/work_steps.rb

```diff
@@ -454,3 +454,9 @@
 Then /^I should not find a list for associations$/ do
   page.should_not have_xpath("//ul[@class=\"associations\"]")
 end
+
+Then /^the work "([^\"]*)" should be deleted$/ do |work|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.